config/initializers/route_validators.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class ValidRegisterApiRoute
```

Comment:
Could we put this in `app/lib` there are a few other constraints in there so would be good to keep them in one place. We should probably have a dedicated directory for them now that I think I about it `app/lib` still doesn't feel quite right

Reply:
moved to `/app/lib/route_validators`